src/config/print/index.md

```diff
@@ -3,20 +3,5 @@
 CUPS (Common Unix Printing System) is the supported mechanism for connecting to
 printers on Void Linux. This section explains how to install and configure CUPS.
 
-Install the CUPS package:
-
-```
-# xbps-install -S cups
-```
-
-Enable the CUPS service:
-
-```
-# ln -s /etc/sv/cupsd /var/service/
-```
-
-Wait until the service is marked available:
-
-```
-# sv status cupsd
-```
+As pre-requisites, install the `cups` package and enable the `cupsd` service.
```

Comment:
Suggest "prerequisites", on the basis of [the Wiktionary entry](https://en.wiktionary.org/wiki/prerequisite).

Reply:
You can probably base that on the entire internet, I'm not sure where that spelling came from.

Fixed!